test/unit/lookups/here_test.rb

```diff
@@ -1,5 +1,4 @@
 # encoding: utf-8
-$: << File.join(File.dirname(__FILE__), "..", "..")
```

Comment:
Only this test failed and when I looked at it to see if I could see any differences I noticed this. I removed it wondering if that might fix the issue but it did not. IDK, the mocking/fixture stuff here is a little mysterious to me but I bet it has something to do with load path, right?